docker/agora/services/app.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  agora-app:
+    image: ghcr.io/sage-bionetworks/agora-app:${AGORA_VERSION:-local}
+    container_name: agora-app
+    restart: always
+    networks:
+      - agora
+    ports:
+      - '5200:5200'
+    env_file:
+      - ../../../apps/agora/app/.env
+    deploy:
+      resources:
+        limits:
+          memory: 200M
+    depends_on:
+      agora-data:
+        condition: service_completed_successfully
```

Comment:
Wait for `agora-data` service to finish prepping the MongoDB before starting `agora-app`